ipn/localapi/localapi.go

```diff
@@ -511,6 +513,35 @@ func (h *Handler) serveDERPMap(w http.ResponseWriter, r *http.Request) {
 	e.Encode(h.b.DERPMap())
 }
 
+// serveShortenExpiry sets the expiry date on the current machine to the
+// specified unix timestamp
```

Comment:
period at end.

and specified how? In the "expiry" parameter. URL parameter or POST parameter? I guess the way you wrote it with FormValue it'll do either.

Reply:
Done